apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -68,9 +68,14 @@
 
 # Used to check Zephyr version installed on the host.
 # We only check two levels of the version.
-ZEPHYR_VERSION = 2.7
+ZEPHYR_VERSION = 3.2
 
-WEST_CMD = default = sys.executable + " -m west" if sys.executable else None
+try:
+    PYTHON_CMD = subprocess.check_output(["which", "python3.8"], encoding="utf-8").strip("\n")
```

Comment:
How about using `shutl.which()` instead of `check_output()`?

Reply:
done